nixos/doc/manual/release-notes/rl-2105.xml

```diff
@@ -23,6 +23,9 @@
      Support is planned until the end of December 2021, handing over to 21.11.
     </para>
    </listitem>
+   <listitem>
+    <para>Linux kernel was updated to the 5.10 LTS series.</para>
```

Comment:
The *default* kernel was. A minor distinction, but that word shows how the power is in the hands of our end-users!

```suggestion
    <para>The default Linux kernel was updated to the 5.10 LTS series, coming from the 5.4 LTS series.</para>
```

See also a previous release note:

 - https://github.com/NixOS/nixpkgs/blob/6839ce74dfe9ceb7a385fb14ea91763c673c138b/nixos/doc/manual/release-notes/rl-2003.xml#L30
 - https://github.com/NixOS/nixpkgs/blob/6839ce74dfe9ceb7a385fb14ea91763c673c138b/nixos/doc/manual/release-notes/rl-2003.xml#L43

Though I guess the actual final form of the release notes will be up to the release managers :).